src/layout/header/Logo.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+const StyledLogo = styled.div`
+  ${({ theme }) => `
+    color: ${theme.palette.text.secondary};
+    font-size: 2rem;
+    grid-area: logo;
+    `}
+`;
```

Comment:
2 alternativer for skrivemåter her:
a)  (min favoritt)
```
const StyledLogo = styled.div`
    color: ${({ theme }) => theme.palette.text.secondary};
    font-size: 2rem;
    grid-area: logo;
`;
```
eller
b)
```
const StyledLogo = styled.div`
    color: ${props => props.theme.palette.text.secondary};
    font-size: 2rem;
    grid-area: logo;
    `}
`;
```

Reply:
stemmer for a